exonum/src/blockchain/builder.rs

```diff
@@ -79,20 +58,11 @@ impl BlockchainBuilder {
     }
 
     /// Add an additional runtime with the specified identifier.
-    pub fn with_additional_runtime(mut self, runtime: impl Into<(u32, Box<dyn Runtime>)>) -> Self {
+    pub fn with_runtime(mut self, runtime: impl Into<(u32, Box<dyn Runtime>)>) -> Self {
```

Comment:
Could you replace `Into<(u32, Box<dyn Runtime>)>` with a more specific trait? E.g.,

```rust
trait WellKnownRuntime: Runtime {
    const ID: u32;
}
```